pygmt/tests/test_clib_loading.py

```diff
@@ -137,7 +137,7 @@ def mock_ctypes(self, monkeypatch):
         """
         monkeypatch.setattr(ctypes, "CDLL", self._mock_ctypes_cdll_return)
 
-    def test_two_broken_libraries(self, mock_ctypes):
+    def test_two_broken_libraries(self, mock_ctypes):  # noqa: ARG002
```

Comment:
If PR #2899 is merged, `# noqa: ARG002` will not be required. So let's wait for #2899 first.